test/unit/controllers/actionbuilders/BatchFileUploadPayloadValidationComposedActionSpec.scala

```diff
@@ -18,40 +18,136 @@ package unit.controllers.actionbuilders
 
 import org.mockito.Mockito.when
 import org.scalatest.mockito.MockitoSugar
+import play.api.http.Status
 import play.api.mvc.AnyContentAsXml
 import play.api.test.FakeRequest
+import uk.gov.hmrc.customs.api.common.controllers.{ErrorResponse, ResponseContents}
 import uk.gov.hmrc.customs.declaration.controllers.actionbuilders.{BatchFileUploadPayloadValidationAction, BatchFileUploadPayloadValidationComposedAction}
 import uk.gov.hmrc.customs.declaration.logging.DeclarationsLogger
-import uk.gov.hmrc.customs.declaration.model._
 import uk.gov.hmrc.customs.declaration.model.actionbuilders.ActionBuilderModelHelper._
 import uk.gov.hmrc.customs.declaration.model.actionbuilders._
+import uk.gov.hmrc.customs.declaration.model.{DocumentType, _}
+import uk.gov.hmrc.customs.declaration.services.DeclarationsConfigService
 import uk.gov.hmrc.play.test.UnitSpec
 import util.ApiSubscriptionFieldsTestData.clientId
 import util.TestData.{conversationId, nonCspRetrievalValues}
 import util.TestXMLData
 
 import scala.concurrent.Future
+import scala.concurrent.duration.Duration
+import scala.xml.Elem
 
 class BatchFileUploadPayloadValidationComposedActionSpec extends UnitSpec with MockitoSugar {
```

Comment:
Can the spacing in all the tests be made consistent so that there is only a blank line between each given/when/then blocks?